### DISCUSSION TITLE
DaWaK Series example

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [SmartDataAnalytics/OpenResearch#257] [General]
* dblp: dblp.uni-trier.de/db/conf/dawak/index.html 23/23
* Wikidata: https://www.wikidata.org/wiki/Q105694634 4/23
* WikiCFP: http://www.wikicfp.com/cfp/program?id=655 9/23
* GND: https://d-nb.info/gnd/1092061126, http://gnd.eurospider.com/s?q=DaWaK&s=&p=1&e=fb&i=sfA&sm=m&so=r 9/23
* OpenResearch: https://www.openresearch.org/wiki/DaWaK 3/23

**Comment by WolfgangFahl** [comment]
I'd love to create an event series completion bot that aligns the different sources see #184. 
Jacob Voß has created a bot that does some comparable linking at https://github.com/gbv/K10PlusBot which might be a good inspiration.

**Comment by nichtich** [comment]
Unfortunately GND does not fully connect DaWaK series and DaWaK events. See [DaWaK GND records](http://gnd.eurospider.com/s?q=DaWaK&s=&p=1&e=fb&i=sfA&sm=m&so=r).

Given an individual event such as http://d-nb.info/gnd/10112318-8 you can look up K10plus PPN with SRU query ["pica.nid=101123188"](http://sru.k10plus.de/opac-de-627!rec=2?version=1.1&operation=searchRetrieve&query=pica.nid%3D101123188&recordSchema=picaxml&startRecord=1&maximumRecords=100), with Catmandu (and [this configuration](https://github.com/pro4bib/pica/blob/master/catmandu.yaml)):

~~~
$ catmandu convert kxpnorm --query "pica.nid=101123188" to JSON | jq -r .[]._id
497772477
~~~

The resulting PPN can be used to query connected publications with a more complex query of form [`pica.1049=$PPN and pica.1045=rel-tt and pica.1001=b`](http://sru.k10plus.de/opac-de-627?version=1.1&operation=searchRetrieve&query=pica.1049%3D497772477%20and%20pica.1045%3Drel-tt%20and%20pica.1001%3Db&recordSchema=picaxml&startRecord=1&maximumRecords=10):

~~~
$ catmandu convert kxpnorm --query "pica.1049=497772477 and pica.1045=rel-tt and pica.1001=b" to pp | picadata -p '021A,017C'
017C $S0$uhttp://www.springerlink.com/openurl.asp?genre=issue&issn=0302-9743&volume=3589$xH$3Volltext$534
017C $uhttp://dx.doi.org/10.1007/11546849$xR$3Volltext$4ZZ$534$ASpringer
017C $uhttp://site.ebrary.com/lib/alltitles/docDetail.action?docID=10528687$xH$3Volltext$534
021A $aData warehousing and knowledge discovery$d7th international conference, DaWaK 2005, Copenhagen, Denmark, August 22 - 26, 2005 ; proceedings$hA. Min Tjoa; Juan Trujillo (ed.)

021A $aData warehousing and knowledge discovery$d7th international conference, DaWaK 2005, Copenhagen, Denmark, August 22 - 26, 2005 ; proceedings$hA. Min Tjoa; Juan Trujillo (ed.)
~~~

In this case one record is the digital edition, the other the printed book.
